html_css/v2/main-course/responsive-css/responsive-css-introduction.md

```diff
@@ -0,0 +1,26 @@
+### Introduction
+"Responsive Design" is an incredibly important concept when it comes to developing for the web. In many ways you are already equipped to create beautiful responsive websites. In this section we're going to fill in the gaps.
+
+### Learning Outcomes
+* You'll learn what 'Responsive Design' is and why it's important.
+
+### Terminology
+The phrase 'Responsive Design' is the most common phrase across the internet to refer to the concept we're talking about here. Unfortunately, it is a little bit misleading, because of the word 'design'. These lessons are not going to focus much on the actual _design_ of responsive websites, but the techniques you can use to implement responsiveness on websites.
+
+The word 'design' refers to making decisions about what kinds of things look nice, crafting a user experience and deciding exactly what a project should look like. There might be some tidbits about that sprinkled throughout these lessons, but it is not the primary focus. HOWEVER, since this phrase has been pretty much universally adopted, it's the phrase we'll use here.
```

Comment:
Is there a reason we're using all caps for HOWEVER here, in favor of bold or italics to call importance to the word, especially given we either in other places?

Reply:
Lol nope.